index.html

```diff
@@ -1,6 +1,24 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <!-- Begin SEO tags-->
+        <meta name="description" content="Hi I'm Lemonbot and it's nice to meet you! I am a Discord bot with a lot of features like moderation, games, fun, etc." />
+		<meta name="keywords" content="discord, bot, fun, games, memes, moderation" />
+		<meta name="og:image" content="https://lemonbot.net/assets/lemonBot.png" />
```

Comment:
Do URLs in the meta tag require a full path? For example would simply `assets/lemonBot.png` work?

Reply:
Yes they require a full path to work.